### ISSUE TITLE
Quaternion integration with IRK

### ISSUE THREAD

**Issue by EveCharbie** [pyomeca/bioptim#241]
The quaternion norm has to be conserved through integration. I think the way to implement it is in this paper.
DOI : 10.1007/s42405-018-0103-6

**Comment by mickaelbegon** [comment]
https://link.springer.com/article/10.1007/s42405-018-0103-6

**Comment by EveCharbie** [comment]
Yes still relevant, it is a missing feature. But I don't need it, so I won't implement it soon.
